pxtpy/converter.ts

```diff
@@ -2433,6 +2434,25 @@ namespace pxt.py {
             if (!syntaxInfo.symbols)
                 syntaxInfo.symbols = []
 
+            // always return global symbols because we might need to check for
+            // name collisions downstream
+            {
```

Comment:
why is this in a block?

Reply:
Used to have an "if",  i'll remove the extra block